src/tools/reconcileLegacyReports.js

```diff
@@ -0,0 +1,3 @@
+import reconcileLegacyReports from '../services/legacyreports';
+
+reconcileLegacyReports();
```

Comment:
Please add a `catch` callback to this, with an implementation that adds an auditlogger call and exits with a non-0 status. The other CLI scripts should give a good example.